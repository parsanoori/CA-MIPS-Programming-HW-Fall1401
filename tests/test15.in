5
692927961
847018819
609347133
1019600207
269514840