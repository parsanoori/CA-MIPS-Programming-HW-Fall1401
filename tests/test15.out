269514840
609347133
692927961
847018819
1019600207